core/Makefile

```diff
@@ -0,0 +1,78 @@
+.DEFAULT_GOAL := get_cbinaries
```

Comment:
Could we please move this file to `core/src/main/c` ?

Reply:
yeah sure